cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsStackRequestHelper.java

```diff
@@ -75,9 +81,10 @@ public DeleteStackRequest createDeleteStackRequest(String cFStackName) {
             keyPairName = awsClient.getExistingKeyPairName(stack.getInstanceAuthentication());
         }
 
-        Collection<Parameter> parameters = new ArrayList<>(asList(
-                new Parameter().withParameterKey("CBUserData").withParameterValue(stack.getImage().getUserDataByType(InstanceGroupType.CORE)),
-                new Parameter().withParameterKey("CBGateWayUserData").withParameterValue(stack.getImage().getUserDataByType(InstanceGroupType.GATEWAY)),
+        Collection<Parameter> parameters = new ArrayList<>();
+        addParameterChunks(parameters, "CBUserData", stack.getImage().getUserDataByType(InstanceGroupType.CORE), CHUNK_COUNT);
+        addParameterChunks(parameters, "CBGateWayUserData", stack.getImage().getUserDataByType(InstanceGroupType.CORE), CHUNK_COUNT);
```

Comment:
Not sure this change is intentional but the original code used `InstanceGroupType.GATEWAY` here.

Reply:
Great catch!